r4/terminology-service-api/Ballerina.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+org = "thiyan"
+name = "terminology_service_api"
+version = "0.1.0"
+distribution = "2201.6.0"
+template=true
+
+[[dependency]]
+org = "ballerinax"
+name = "health.fhir.r4"
+version = "3.0.1"
+repository = "local"
```

Comment:
The automated build is failing due to the r4 version I used here. I'm using the next version of the r4 and a local build artifact because this terminology library implementation is not released yet.
PR:  https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/pull/19